nss-day-cohort-39/tabloid-full-stack-velours
# Delete Tag

### Changes
- Refactored TagRepo to remove the relationship in the PostTags join table using Entity Framework

### Testing
- Login as an Admin and view the details of any Post
- Either find or add a tag that is associated with the post via Tag Manager
- Click the Tags link at the top
- Remove the tag that you either created or found
- Go back to the Post Details and confirm the tag does not appear on the post
- Confirm that the tag does not appear in the Tag Manager
